source/plugin/bukkit-sponge-compare.rst

```diff
@@ -0,0 +1,680 @@
+event.server.ServiceEvent
+event.server.ServiceRegisterEvent
+event.server.ServiceUnregisterEvent
+event.vehicle.VehicleBlockCollisionEvent
+event.vehicle.VehicleCollisionEvent
+event.vehicle.VehicleCreateEvent
+event.vehicle.VehicleDamageEvent
+event.vehicle.VehicleDestroyEvent
+event.vehicle.VehicleEnterEvent
+event.vehicle.VehicleEntityCollisionEvent
+event.vehicle.VehicleEvent
+event.vehicle.VehicleExitEvent
+event.vehicle.VehicleMoveEvent
+event.vehicle.VehicleUpdateEvent
+event.weather.LightningStrikeEvent
+event.weather.ThunderChangeEvent
+event.weather.WeatherChangeEvent                       event.world.ChangeWorldWeatherEvent
+event.weather.WeatherEvent
+event.world.ChunkEvent                                 event.world.chunk.TargetChunkEvent
+event.world.ChunkLoadEvent                             event.world.chunk.LoadChunkEvent
+event.world.ChunkPopulateEvent                         event.world.chunk.PopulateChunkEvent
+event.world.ChunkUnloadEvent                           event.world.chunk.UnloadChunkEvent
+event.world.PortalCreateEvent
+event.world.SpawnChangeEvent
+event.world.StructureGrowEvent
+event.world.WorldEvent                                 event.world.TargetWorldEvent
+event.world.WorldInitEvent                             event.world.ConstructWorldEvent
+event.world.WorldLoadEvent                             event.world.LoadWorldEvent
+event.world.WorldSaveEvent                             event.world.SaveWorldEvent
+event.world.WorldUnloadEvent                           event.world.UnloadWorldEvent
+event.Cancellable                                      event.Cancellable
+event.Event                                            event.Event
+event.EventException
+event.EventHandler                                     event.Listener
+event.EventPriority                                    event.Order
+event.HandlerList
+event.Listener
+generator.BlockPopulator
+generator.ChunkGenerator
+help.GenericCommandHelpTopic
+help.HelpMap
+help.HelpTopic
+help.HelpTopicComparator
+help.HelpTopicFactory
+help.IndexHelpTopic
+inventory.meta.BannerMeta
+inventory.meta.BlockStateMeta
+inventory.meta.BookMeta
+inventory.meta.EnchantmentStorageMeta
+inventory.meta.FireworkEffectMeta
+inventory.meta.FireworkMeta
+inventory.meta.ItemMeta
+inventory.meta.LeatherArmorMeta
+inventory.meta.MapMeta
+inventory.meta.PotionMeta
+inventory.meta.Repairable
+inventory.meta.SkullMeta
+inventory.AnvilInventory
+inventory.BeaconInventory
+inventory.BrewerInventory
+inventory.CraftingInventory
+inventory.DoubleChestInventory
+inventory.EnchantingInventory
+inventory.EntityEquipment
+inventory.EquipmentSlot
+inventory.FurnaceInventory
+inventory.FurnaceRecipe
+inventory.HorseInventory
+inventory.Inventory
+inventory.InventoryHolder
+inventory.InventoryView
+inventory.ItemFactory
+inventory.ItemFlag
+inventory.ItemStack
+inventory.MerchantInventory
+inventory.PlayerInventory
```

Comment:
`HumanInventory`, though this should change to `PlayerInventory` soon (probably in the bleeding branch already in API).